pmu.cpp

```diff
@@ -94,7 +94,69 @@ void PMU::reset()
 	const int result = ioctl(_fd, PERF_EVENT_IOC_RESET, 0);
 	if (result == -1)
 	{
-		throw std::runtime_error("Failed to reset PMU counter: " + std::to_string(errno));
+		HWCPIPE_LOG("Failed to reset PMU counter: %s", std::to_string(errno).c_str());
```

Comment:
This will be printed frequently if `adb shell setprop security.perf_harden 0` has not been run

Reply:
The problem is that `PMUCounter::start()` doesn't have a way to tell if the counters are working or not (by the way there are two counters missing there, we should add them).

We already print an error message on `PMU::open([...])`, so it's probably not necessary to keep logging on every attempt to reset.

My suggestion would be to change `PMU::reset()` to return a `bool` indicating success, instead of logging failure.

Reply:
Done